eslint-plugin-expensify/tests/prefer-constructors-with-props.test.js

```diff
@@ -0,0 +1,54 @@
+const RuleTester = require('eslint').RuleTester;
+const rule = require('../prefer-constructors-with-props');
+const message = require('../CONST').MESSAGE.PREFER_CONSTRUCTORS_WITH_PROPS;
+
+const ruleTester = new RuleTester({
+    parserOptions: {
+        ecmaVersion: 6,
+        sourceType: 'module',
+    },
+});
+
+const goodExample = `
```

Comment:
Let's add a good exmple like:

```js
class Random {
    constructor() {
    }
}
```

Reply:
sure, i'll add it and am sure it's gonna fail.